README.md

```diff
@@ -82,7 +87,12 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
+ ::= "the" | "the" 
+ ::= |
+ ::= "lazy" | "smelly" 
+ ::= "dog" | "cat"
+ ::= "ran" | "ate"
+ ::= "slowly" | "noisily"
```

Comment:
# 0

A BNF grammar is rules of the form

    <nonterminal> ::= list of terminals or non terminals

There will be a starting nonterminal (in this case <sentence>) from which the rest of the productions can generate the grammar.